### PR TITLE
Add support for Opt-In Split Tunnel Overlapping IPs

### PR DESCRIPTION
**Pull request by da-cf**
## Description
Add two new fields to the device settings policy. This is to add API support for the Opt-In Split Tunnel Overlapping IPs feature:
1. `lan_allow_minutes` (uint) used to set the amount of time a user can access a LAN subnet.
2. `lan_allow_subnet_size` (uint) the allowed subnet size for the lan 

## Has your change been tested?
Added to existing device settings policy tests.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.
- [X] This change is using publicly documented in [cloudflare/api-schemas](https://github.com/cloudflare/api-schemas) 
      and relies on stable APIs.

[1]: https://help.github.com/articles/closing-issues-using-keywords/

### CODE REVIEW

File: devices_policy.go [hunk 1]
```diff
@@ -57,6 +57,8 @@ type DeviceSettingsPolicy struct {
 	Default             bool              `json:"default"`
 	ExcludeOfficeIps    *bool             `json:"exclude_office_ips"`
 	Description         *string           `json:"description"`
+	LanAllowMinutes     *uint             `json:"lan_allow_minutes"`
```
Comment (jacobbednarz) [inline devices_policy.go]: given LAN is an acronym, we should uppercase the usage in all these cases much like we would for HTTP or TCP (`Lan_` => `LAN_`)

Comment (da-cf) [reply devices_policy.go]: Hey Jacob, thank you for the prompt review.

While I agree that this should have been `LAN_` from the start, most, if not all, of the teams working on this have already standardized with the `lan_allow_minutes` json field name.
I don't know if implementing this change is worth having a handful of people update their field name, but I'm interested to hear your thoughts.

Thanks again :)

Comment (jacobbednarz) [reply devices_policy.go]: 👋 my point is only referring to the struct name here which is what we are adding support for here; the json key is perfect.

Comment (da-cf) [reply devices_policy.go]: Oh that's perfect, then. Will implement :)

**Pull request merged**